public/vendor/plugins/sortt/sortt.js

```diff
@@ -0,0 +1,48 @@
+/*
+  Sortt
+
+  @License MIT
+  @Author 6543
+  @Repository https://gitea.com/6543/gitea_sortt
+  @Version 1.0
+
+*/
+
+//creat global function with main routine
+window.sortt=function(normsort,revsort,isdefault){
+  //sortTable [normsort] (revsort) (isdefault)
+
+  //normsort is needet
+  if (typeof normsort  === 'undefined') return false;
+
+  //default values of optinal parameters
+  if (typeof revsort  === 'undefined') revsort = "";
+  if (typeof isdefault  === 'undefined') isdefault = false;
+  if (!(typeof isdefault  === 'boolean')) isdefault = false;
```

Comment:
Just to help with readability although this is more preference.
```suggestion
  if (typeof isdefault  !== 'boolean') isdefault = false;
```